### PR TITLE
Overhaul of the chapter about image handling.

### PR DESCRIPTION
**Pull request by jensens**
Improves:
- content image chapter

Changes proposed in this pull request:
- overhauled to match with dx and Plone 5

### CODE REVIEW

File: develop/plone/images/content.rst [hunk 1]
```diff
@@ -11,28 +11,41 @@
 Introduction
 ============
 
-Plone supports image content in two forms:
+Plone supports image content in several forms:
 
-* As stand-alone content type, images will be visible in the sitemap. This is
-  the case for the default ``Image`` content type, but you can create custom
-  content types with similar properties.
+standlone content type
```
Comment (polyester) [inline develop/plone/images/content.rst]: typo...

Comment (jensens) [inline develop/plone/images/content.rst]: stand-alone or standalone?

Comment (polyester) [inline develop/plone/images/content.rst]: most sources say stand-alone, but now it says standlone ;-)

**Comment by polyester** [comment]
thanks! much needed overhaul.

**Comment by svx** [comment]
@jensens yes thanks a lot ! Next push to docs.plone.org will be on Friday

**Pull request merged**